brainglobe_atlasapi/atlas_generation/atlas_scripts/allen_human.py

```diff
@@ -11,14 +10,14 @@
 import urllib3
 from allensdk.core.structure_tree import StructureTree
 from brainglobe_utils.IO.image import load_nii
+from openminds_allen_human_wrapup import wrapup_atlas_to_openminds
```

Comment:
```suggestion
```
Any changes related to attempts to tackle #356 should not be in this PR, as they are unrelated.